cdap-docs/developers-manual/source/building-blocks/datasets/system-custom.rst

```diff
@@ -101,7 +101,14 @@ To create a Dataset of type ``UniqueCountTable``, add the following into the App
 
 You can also pass ``DatasetProperties`` as a third parameter to the ``createDataset`` method.
 These properties will be used by embedded Datasets during creation and will be available via the
-``DatasetSpecification`` passed to the Dataset constructor.
+``DatasetSpecification`` passed to the Dataset constructor. For example, to create a 
```

Comment:
I think this belongs into the Table API section. Aren't the properties accepted by the Table part of its API?